modules/gmc-agent/src/test/java/org/gridgain/config/WebSocketConfig.java

```diff
@@ -44,6 +45,11 @@
         registry.addEndpoint(AGENTS_PATH);
     }
 
+    /** {@inheritDoc} */
+    @Override public void configureWebSocketTransport(WebSocketTransportRegistration registration) {
+        registration.setMessageSizeLimit(128 * 1024);
```

Comment:
Why we need to increase chunk size?

Reply:
In some cases in tests we have very huge metric message.